src/Peering/Peering/help/Set-AzPeeringDirectConnectionObject.md

```diff
@@ -32,7 +32,13 @@ Set-AzPeeringDirectConnectionObject -InputObject <PSDirectConnection> -SessionPr
 
 ### Md5Authentication
 ```
-Set-AzPeeringDirectConnectionObject -InputObject <PSDirectConnection> [-MD5AuthenticationKey <String>]
+Set-AzPeeringDirectConnectionObject -InputObject <PSDirectConnection> -MD5AuthenticationKey <String>
```

Comment:
@rileymckenna what's the reason for making this mandatory now? What was the behavior previously if user's didn't provide the `-MD5AuthenticationKey` parameter?

Reply:
it would set the value to null, now they have to specify null. 
`this.InputObject.BgpSession.Md5AuthenticationKey = this.MD5AuthenticationKey;`